### ISSUE TITLE
Implement updated name_spec for displayForm

### ISSUE THREAD

**Issue by arcadiafalcone** [sul-dlss/cocina-models#479]

**Comment by arcadiafalcone** [comment]
@justinlittman Let me know if I can provide any assistance with moving this ticket forward. I don't think any data was lost in migration but it will be inconsistently structured in Cocina (and incorrect in MODS), and will need analysis/remediation to align with preferred data structures.

**Comment by justinlittman** [comment]
@arcadiafalcone No one is currently working on this ticket. @ndushay is this in scope for maintenance workcycle? Can it be added to the board?

**Comment by ndushay** [comment]
@justinlittman yes, it will be on the current workcycle.  No, it is not done.  No, I am not likely to be able to revisit this;  yes, Justin Littman, it would be great if you could work with Arcadia to carry this to completion.  I have a PR up that is a draft but not sure it's up to Arcadia's latest specs.  Happy to consult.

**Comment by ndushay** [comment]
The actions needed for this ticket are:
- talk to Arcadia to determine if the unimplemented specs for this are what she wants.  If they aren't, ask Arcadia to revise those specs.
- determine if the existing PR that I wrote fulfills Arcadia's specs, and if so, and if they are passing, yay, you're done - get the PR reviewed and merged.  Otherwise revise that branch or start a new one and fulfill Arcadia's specs.  Not sure how to test this is a broader view since we have trashed the caches, AFAIK - ask @justinlittman.
